packages/api-specification/src/new-window-api.html

```diff
@@ -38,7 +38,7 @@
 <h3>Method Signature</h3>
 
 <pre>
-window.open(url, name, features)
+ssf.Window(url, name, features)
```

Comment:
I think we'll need to consider whether the API will follow the factory pattern (used here), or the constructor pattern. For example:

```js
// Factory
var newWindow = ssf.window();

// Constructor
var newWindow = new ssf.Window();
```

From what I can see on the Wiki, there seems to be a preference for the constructor pattern, but I'm happy to use the factory pattern, get some feedback, then change if required.

However, if we are going to stick with the factory pattern for now, I think the casing should be changed. PascalCase for classes and constructors and camelCase for everything else is a common convention, so `ssf.Window()` => `ssf.window()`